test/factories/users.rb

```diff
@@ -8,7 +8,7 @@
     sequence(:email) { |n| "sanyi_#{n}@example.org" }
     sequence(:cell_phone) { |n| "66677788#{n}" }
     sequence(:neptun) do |n|
-      random_character = (n + 48).chr
+      random_character = (n + 48).chr(Encoding::UTF_8)
```

Comment:
Travis build failed, with the following output:
- Failure/Error: let(:user)        { create(:user) }
- ActiveRecord::StatementInvalid: PG::CharacterNotInRepertoire: ERROR:  invalid byte sequence for encoding "UTF8": 0x80

I think this occurred because Integer.chr does not enforce encoding by default, only if encoding is specified as a parameter.